docs/learning/ui/core/Settings.md

```diff
@@ -0,0 +1,150 @@
+# Settings
+
+The [SettingsManager]($ui-core) allows the registration of [SettingsProvider]($ui-core) classes to provide the settings to display with the [SettingsContainer]($ui-core) component. In an application that employees App-UI, the [SettingsModalFrontstage]($ui-framework) frontstage will display the SettingsContainer and its entries.

```

Comment:
Did you mean `employs` instead of `employees`?

Also, are we referring to it as App UI or App-UI?  Just wondering for consistency across the docs.

Reply:
I talked to Nancy and we are going to use 'App UI' through out .... so I will also update 'AppUI' to 'App UI' in other places in the docs.

Reply:
Okay, sounds good to me.

Reply:
Updated to 'App UI' and fixed misspelling.